Makefile

```
@@ -177,39 +183,55 @@ start-test-frontend: ## Start Test Volto Frontend
 	$(MAKE) -C "./api/" start-test
 
 .PHONY: stop-backend-docker-guillotina
 stop-backend-docker-guillotina:
 	docker-compose -f g-api/docker-compose.yml down
 
-.PHONY: test-acceptance-server
-test-acceptance-server:
+.PHONY: start-test-acceptance-server test-acceptance-server
+start-test-acceptance-server test-acceptance-server: ## Start Test Acceptance Server Main Fixture (docker container)
 	docker run -i --rm -e ZSERVER_HOST=0.0.0.0 -e ZSERVER_PORT=55001 -p 55001:55001 -e ADDONS='$(KGS) plone.app.robotframework plone.app.contenttypes' -e APPLY_PROFILES=plone.app.contenttypes:plone-content,plone.restapi:default,plone.volto:default-homepage -e CONFIGURE_PACKAGES=plone.app.contenttypes,plone.restapi,plone.volto,plone.volto.cors $(DOCKER_IMAGE) ./bin/robot-server plone.app.robotframework.testing.PLONE_ROBOT_TESTING
 
-.PHONY: test-acceptance-server-multilingual
-test-acceptance-server-multilingual:
+.PHONY: start-test-acceptance-server-multilingual test-acceptance-server-multilingual
+start-test-acceptance-server-multilingual test-acceptance-server-multilingual: ## Start Test Acceptance Server Multilingual Fixture (docker container)
 	docker run -i --rm -e ZSERVER_HOST=0.0.0.0 -e ZSERVER_PORT=55001 -p 55001:55001 -e ADDONS='$(KGS) plone.app.robotframework plone.app.contenttypes' -e APPLY_PROFILES=plone.app.contenttypes:plone-content,plone.restapi:default,plone.volto:multilingual -e CONFIGURE_PACKAGES=plone.app.contenttypes,plone.restapi,plone.volto,plone.volto.cors $(DOCKER_IMAGE) ./bin/robot-server plone.app.robotframework.testing.PLONE_ROBOT_TESTING
 
-.PHONY: test-acceptance-server-workingcopy
-test-acceptance-server-workingcopy:
+.PHONY: start-test-acceptance-server-workingcopy test-acceptance-server-workingcopy
+start-test-acceptance-server-workingcopy test-acceptance-server-workingcopy : ## Start Test Acceptance Server WorkingCopy Fixture (docker container)
 	docker run -i --rm -e ZSERVER_HOST=0.0.0.0 -e ZSERVER_PORT=55001 -p 55001:55001 -e ADDONS='$(KGS) plone.app.robotframework plone.app.contenttypes' -e APPLY_PROFILES=plone.app.contenttypes:plone-content,plone.restapi:default,plone.app.iterate:default,plone.volto:default-homepage -e CONFIGURE_PACKAGES=plone.app.contenttypes,plone.restapi,plone.volto,plone.volto.cors $(DOCKER_IMAGE) ./bin/robot-server plone.app.robotframework.testing.PLONE_ROBOT_TESTING
 	# ZSERVER_PORT=55001 CONFIGURE_PACKAGES=plone.app.contenttypes,plone.restapi,plone.app.iterate,plone.volto,plone.volto.cors APPLY_PROFILES=plone.app.contenttypes:plone-content,plone.restapi:default,plone.app.iterate:default,plone.volto:default-homepage ./api/bin/robot-server plone.app.robotframework.testing.PLONE_ROBOT_TESTING
 
-.PHONY: test-acceptance-server-coresandbox
-test-acceptance-server-coresandbox:
+.PHONY: start-test-acceptance-server-coresandbox test-acceptance-server-coresandbox
+start-test-acceptance-server-coresandbox test-acceptance-server-coresandbox: ## Start Test Acceptance Server CoreSandbox Fixture (docker container)
```

target dependencies are written after the double colon
```suggestion
start-test-acceptance-server-coresandbox: test-acceptance-server-coresandbox ## Start Test Acceptance Server CoreSandbox Fixture (docker container)
```

---

In fact, it was not meant to be a dependency, but an alias. I wanted to give it a more meaningful name. Maybe it's better to deprecate the old name right away... but add-on generator relied on these names.